### DISCUSSION TITLE
Welcome to hair-segmentation Discussions!

### DISCUSSION THREAD

**Discussion by thangtran480** [thangtran480/hair-segmentation#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shubham0204** [comment]
Hi @thangtran480 
I saw that you need to build an Android app which can run the hair segmentation model. I would like to contribute the Android app, as I have made a similar project for face segmentation. See [shubham0204/MLKit_Selfie_Segmentation_Android](https://github.com/shubham0204/MLKit_Selfie_Segmentation_Android). Let me know your opinion.

**Reply by thangtran480** [reply]
@shubham0204 thanks for visiting my project. I uploaded file mobile for python and tflite (mobile), you can use it to deploy into your project
